test/cypress/support/commands.ts

```diff
@@ -37,8 +38,10 @@ const getCommonHeaders = () => ({
 declare global {
     namespace Cypress {
         // NOTE: necessary for extending the Cypress API
-        // eslint-disable-next-line @typescript-eslint/no-empty-interface
-        export interface Chainable extends GetCypressChainableFromCommands<typeof commands> {}
+        export interface Chainable extends GetCypressChainableFromCommands<typeof commands> {
+            // typing for cypress-get-table
+            getTable: () => Cypress.Chainable;
```

Comment:
Optional:

We could enhance the type safety here by specifying the subject type:

```suggestion
            getTable: () => Cypress.Chainable<Record<string, any>[]>;
```

Reply:
Cool, done.